tests/utils/tests/test_schemapi.py

```diff
@@ -377,9 +378,7 @@ def test_schema_validation_error():
     assert isinstance(the_err, SchemaValidationError)
     message = str(the_err)
 
-    assert message.startswith("Invalid specification")
-    assert "test_schemapi.MySchema->a" in message
-    assert "validating {!r}".format(the_err.validator) in message
```

Comment:
This was all part of the redundant info that is not removed from the error message. Maybe it it redundant to have both the lines below actually.

Reply:
Yup these two lines were testing the same message so removing one.